reference.md

```diff
@@ -745,11 +745,11 @@ The request must have the following query parameters.
 
 Field Name | Presence | Type | Description
 ---|---|---|---
-`pickup_lat` | REQUIRED | Latitude | Latitude of the location where the user will be picked-up. 
-`pickup_lon` | REQUIRED | Longitude | Longitude of the location where the user will be picked-up. 
+`pickup_lat` | REQUIRED | Latitude | Latitude of the location where the user will be picked-up.
+`pickup_lon` | REQUIRED | Longitude | Longitude of the location where the user will be picked-up.
 `drop_off_lat` | Conditionally REQUIRED | Latitude | Latitude of the location where the user will be dropped off. Required if `drop_off_lon` is provided. FORBIDDEN otherwise.
 `drop_off_lon` | Conditionally REQUIRED | Longitude | Longitude of the location where the user will be dropped off. Required if `drop_off_lat` is provided. FORBIDDEN otherwise.
-`brand_id` | Conditionally REQUIRED | ID | Brand ID from `service_brands.json` to define the wait time is requested for which brand. REQUIRED if more than one service brand is available.  
+`brand_id` | Conditionally REQUIRED | ID |  Array | Array of service brand Ids defined in `service_brands.json`. REQUIRED if more than one service brand is available.  
```

Comment:
@E-Baird @gcamp 
You mentionned you want brand_id as optionnal, but the original spec stated "REQUIRED if more than one service brand is available.", so I left it as is. I will let Transit change the original spec.

Reply:
Good catch. I'll discuss with people on our side and if we need to change this, we can do it later in another PR. It shouldn't affect Taxi MTL as we will make sure to always send you a value here.